app/resources/coins/coin.rb

```diff
@@ -5,11 +5,11 @@ class Coin < Resource
   include Valkyrie::Resource::AccessControls
   attribute :member_ids, Valkyrie::Types::Array
   attribute :member_of_collection_ids
+  attribute :numismatic_accession_id
 
   # descriptive metadata
   attribute :numismatic_citation, Valkyrie::Types::Array.of(NumismaticCitation).meta(ordered: true)
   attribute :coin_number, Valkyrie::Types::Anything
-  attribute :number_in_accession, Valkyrie::Types::Integer
```

Comment:
I think we still need this property — my understanding is that items within an accession are numbered, and this lets you say "this coin is item #5 in that accession"

Reply:
Ah, right. I mean to remove `accession_number` which was the way that accessions were linked to coins before this PR. I'll switch that around and `number_in_accession` back and get rid of `accession_number`.